css/pages/index.css

```diff
@@ -1,16 +1,52 @@
+* {
+  box-sizing: border-box;
+}
+
 .container {
   max-width: 750px;
   text-align: center;
   font-size: 1.5em;
 }
 
+.cards-title {
+  margin-bottom: 2rem;
+}
+
+.cards {
+  display: flex;
+  justify-content: space-around;
+}
+
+.card {
+  border: 1px solid #f3efef;
+  box-shadow: 0 1.5px 10px 0 rgba(0, 0, 0, 0.14);
+  border-radius: 2px;
+  width: 14rem;
+  padding: 18px 15px;
+}
+
+.card a {
+  font-weight: 600;
+  font-size: 20px;
```

Comment:
I see we've used `rem` for font-size at some places. Are we not using rem for font-size here?
If a user changes the default font-size for their browser, our code wouldn't adapt to their new settings. Specifying `px` would override their changes.

Please read this short answer: https://css-tricks.com/is-it-better-to-use-ems-rems-than-px-for-font-size/

Reply:
> I see we've used `rem` for font-size at some places. Are we not using rem for font-size here?
> If a user changes the default font-size for their browser, our code wouldn't adapt to their new settings. Specifying `px` would override their changes.
> 
> Please read this short answer: https://css-tricks.com/is-it-better-to-use-ems-rems-than-px-for-font-size/

Okay I will replace px with rem for font-size